js/data/array_query.js

```diff
@@ -213,7 +222,7 @@ const SortIterator = Iterator.inherit({
             const rule = this.rules[i];
             const xValue = rule.getter(x);
             const yValue = rule.getter(y);
-            const compare = rule.compare || defaultCompare;
+            const compare = rule.compare || ((xValue, yValue) => defaultCompare(xValue, yValue, rule.compareOptions));
```

Comment:
Minor comment.

Let's create ```getDefaultCompare``` function instead of an inline anonymous function here
E.g
```typescript
const getDefaultCompareFunc = (compareOptions) => 
  (xValue, yValue) => defaultCompare(xValue, yValue,compareOptions);
```

And use it here

Reply:
ok